wrapanapi/scvmm.py

```diff
@@ -40,11 +40,17 @@ class SCVMMSystem(WrapanapiAPIBaseVM):
     def __init__(self, **kwargs):
         super(SCVMMSystem, self).__init__(kwargs)
         self.host = kwargs["hostname"]
+        self.port = kwargs.get("winrm_port", 5985)
+        self.scheme = kwargs.get("winrm_scheme", "http")
         self.user = kwargs["username"]
         self.password = kwargs["password"]
         self.domain = kwargs["domain"]
         self.provisioning = kwargs["provisioning"]
-        self.api = winrm.Session(self.host, auth=(self.user, self.password))
+        self.api = winrm.Session(
+            '{scheme}://{host}:{port}'.format(scheme=self.scheme, host=self.host, port=self.port),
+            auth=(self.user, self.password),
+            server_cert_validation='ignore',
```

Comment:
Let's make this a boolean kwarg with a default to ignore. like `='<string for validation' if kwargs.get('cert_validation') else 'ignore'`